aiogram/fsm/storage/base.py

```diff
@@ -64,6 +64,17 @@ async def get_data(self, key: StorageKey) -> Dict[str, Any]:
         """
         pass
 
+    async def get_value(self, key: StorageKey, data_key: str) -> Any:
+        """
+        Get selected value by key in current data
+
+        :param key: storage key
+        :param data_key: key of selected data
+        :return: value of current data by key
+        """
+        current_data = await self.get_data(key=key)
+        return current_data[data_key]
```

Comment:
What about using method `get` with default value?
```python
return current_data.get(data_key, default=default)
```

The use of this method (`get_value`) is not very clear, so I do not understand whether there should be an error or a default value, it is necessary to further explain the use case so that I understand what should be here.